dist/37.index.js

```diff
@@ -0,0 +1,452 @@
+"use strict";
```

Comment:
Is this file needed? What does it do?

Reply:
Oh, sorry, I just did `git add dist` and didn't notice. Have no idea, but when I did `rm -r dist && npm run build`, it was not re-created, so it doesn't seem like it's necessary indeed! Removed in 4a65b1d 👍